examples/textinput.py

```diff
@@ -6,14 +6,15 @@
 Shows how to use the TEXTEDITING and TEXTINPUT events.
 """
 import sys
+import os
 
 import pygame
 import pygame as pg
 import pygame.freetype as freetype
 
-# Version check
-if pg.get_sdl_version() < (2, 0, 0):
-    raise Exception("This example requires pygame 2.")
+""" This is very important
+    If not added the candidate list will not show """
```

Comment:
minor nitpick: This can just be regular `#` comments. `"""` are generally reserved for doc strings and multiline strings

Reply:
Sure.